iterating-over-data/practice-iterating-over-data.md

```diff
@@ -0,0 +1,465 @@
+# Practice: Iterating Over Data
+
+## Directions
+
+Complete all questions below.
+
+## Practice
+
+<!-- Question 1 -->
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: multiple-choice
+* id: EJew-RW0t
+* title: Which Range is Correct?
+
+##### !question
+
+Given a function called `stopwatch` that takes in the argument `22`, which of the following loops would  correctly start at `1` and increment all the way up to *and include* `22`?
```

Comment:
Dropped accidental double-space, remove "correctly"-- the question is inherently asking for the correct solution

```suggestion
Given a function called `stopwatch` that takes in the argument `22`, which of the following loops would start at `1` and increment all the way up to *and include* `22`?
```